samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -79,6 +80,14 @@ for (let i = 0; i < 60; i++) {
 
 const ContextualPopupButton = ContextualPopupDecorator(Button);
 
+const isRTL = () => {
```

Comment:
We can use `I18nContextDecorator` rather than use a hard coded locale.
Please refer to the https://enactjs.com/docs/developer-guide/i18n/#accessing-i18ndecorator-context-properties.
You also can refer to the sample https://github.com/enactjs/sandstone/blob/develop/samples/sampler/stories/default/PopupTabLayout.js#L195

Reply:
Suggested changes.